babi_multiprocess.diff

```diff
@@ -19,11 +19,27 @@ index a21e9e63e8..bd22d3b073 100644
              raise
  
      def add(self, cls, type='model'):
-diff --git a/tryton/trytond/trytond/model/modelsql.py b/trytond/trytond/model/modelsql.py
-index 889916371c..ae51a5d433 100644
+diff --git a/tryton/trytond/trytond/model/modelsql.py b/tryton/trytond/trytond/model/modelsql.py
+index a21e9e63e8..a204c6df01 100644
 --- a/tryton/trytond/trytond/model/modelsql.py
 +++ b/tryton/trytond/trytond/model/modelsql.py
-@@ -1209,8 +1209,14 @@ class ModelSQL(ModelStorage):
+@@ -753,7 +753,14 @@ class ModelSQL(ModelStorage):
+             if '.' in field_name:
+                 field_name, field_related = field_name.split('.', 1)
+                 fields_related[field_name].add(field_related)
+-            field = cls._fields[field_name]
++            try:
++                field = cls._fields[field_name]
++            except KeyError as err:
++                if 'babi_' in str(err):
++                    fields_names.remove(field_name)
```

Comment:
field_names is a parameter to the method so it should not be modified.

We should:

```
fields_names = fields_names[:]
fields_names.remove(field_name)
```